stamina-testkit/src/main/scala/stamina/testkit/StaminaTestKit.scala

```diff
@@ -0,0 +1,85 @@
+package stamina
+package testkit
+
+import scala.util._
+import scala.annotation.tailrec
+
+trait StaminaTestKit { self: org.scalatest.WordSpecLike ⇒
```

Comment:
I remember we discussed whether to also support specs2 in the future. I think it would be better to prepare for that eventuality by renaming this trait to `StaminaScalatestTestkit` or something similar to leave the door open for a second implementation.